features/breed_detail/src/main/java/com/example/catbreeds/breed_detail/BreedDetailScreen.kt

```diff
@@ -86,67 +99,88 @@ fun BreedDetailScreen(
                 modifier = Modifier
                     .fillMaxSize()
                     .padding(paddingValues)
-                    .padding(16.dp)
-                    .verticalScroll(rememberScrollState()),
-                verticalArrangement = Arrangement.spacedBy(16.dp)
+                    .verticalScroll(scrollState),
+                verticalArrangement = Arrangement.spacedBy(AppDimensions.DetailsVerticalSpacing)
             ) {
                 // Cat image
                 AsyncImage(
                     model = "https://cdn2.thecatapi.com/images/${breed.reference_image_id}.jpg",
                     contentDescription = "Image of ${breed.name}",
                     modifier = Modifier
                         .fillMaxWidth()
-                        .clip(RoundedCornerShape(8.dp)),
+                        .padding(horizontal = AppDimensions.ScreenPadding)
+                        .clip(RoundedCornerShape(AppDimensions.CardCornerRadius)),
                     contentScale = ContentScale.Crop,
-                    placeholder = painterResource(id = R.drawable.ic_menu_report_image),
-                    error = painterResource(id = R.drawable.ic_menu_close_clear_cancel)
+                    placeholder = painterResource(id = R.drawable.ic_cat_placeholder),
+                    error = painterResource(id = R.drawable.ic_cat_error)
                 )
-                // Name and origin
+
+                // Origin / Life Expectancy
                 Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    horizontalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = AppDimensions.ScreenPadding),
+                    horizontalArrangement = Arrangement.spacedBy(AppDimensions.InterItemSpacing),
                     verticalAlignment = Alignment.CenterVertically
                 ) {
-                    Column(modifier = Modifier.weight(1f)) {
-                        Text(
-                            text = breed.name,
-                            fontSize = 24.sp,
-                            fontWeight = FontWeight.Bold
-                        )
-                        Text(
-                            text = "Origin: ${breed.origin}",
-                            fontSize = 16.sp,
-                            color = MaterialTheme.colorScheme.onSurfaceVariant
-                        )
-                    }
+                    StatCard(
```

Comment:
try to keep the params names (specially when there are multiple ones). 
Also for the weight, what do you think of creating a Constants file where you can set this recurring values ? So there you will set now the 
`DEFAULT_WEIGHT = 1f`

```suggestion
                    StatCard(
                        modifier =  Modifier.weight(DEFAULT_WEIGHT)
                        label = "Origin",
                        value = breed.origin,
                    )
```

Reply:
Already added the params names. As for the weight:
- I defined the DefaultWeight on my existing Dimensions.kt
- I used UpperCamelCase instead of 'FULL_SNAKE_CASE' to fit with the rest of the constants. If you think I should change that on all of them just let me know, should be a quick refactor